p2p/chain.py

```diff
@@ -625,55 +555,17 @@ class RegularChainSyncer(FastChainSyncer):
         elif isinstance(cmd, eth.GetBlockHeaders):
             await self._handle_get_block_headers(peer, cast(Dict[str, Any], msg))
         elif isinstance(cmd, eth.GetBlockBodies):
-            await self._handle_get_block_bodies(peer, cast(List[Hash32], msg))
+            await handle_get_block_bodies(
+                self.db, peer, cast(List[Hash32], msg), self.logger, self.cancel_token)
         elif isinstance(cmd, eth.GetReceipts):
-            await self._handle_get_receipts(peer, cast(List[Hash32], msg))
+            await handle_get_receipts(
```

Comment:
In conjunction with my previous comment, these would now be `await self.handler.get_receipts(peer, ...)`